### ISSUE TITLE
[DynamicPage]: Elements must only use permitted ARIA attributes

### ISSUE THREAD

**Issue by raoakash01** [SAP/ui5-webcomponents-react#7833]
### Describe the bug

Dynamic Page has unwanted Aria attribute which is not needed and we have received an accessibility bug regarding this 
here in the example and the screen shot we can see that we have "aria-labelledby" which is not required.
<img width="1798" height="840" alt="Image" src="https://github.com/user-attachments/assets/05d56681-d037-4d22-a6ca-f678362d6566" />

### Isolated Example

https://stackblitz.com/edit/github-gaqjtizs?file=src%2FApp.tsx,package.json

### Reproduction steps

1. run the shared example in stackblitz
2. inspect it and we can check it in the element section
3. or we can also use a tool axe Dev Tool and scan the page for accessibility 

### Expected Behaviour

aria-labelledby attribute cannot be used on a span with no valid role attribute.

### UI5 Web Components for React Version

2.15.0

### UI5 Web Components Version

2.15.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @raoakash01 

I will close this issue as there is already another one covering this topic. If you have any questions or more information, please address them in the [original issue](https://github.com/UI5/webcomponents/issues/12466).
